highfidelity/hifi
# Fix and expose Particle Entity emitRadiusStart

https://highfidelity.manuscript.com/f/cases/19496/Particle-entity-emitRadiusStart-not-exposed-and-not-working

Test plan:
- "Emit Radius Start" slider should show up in particle properties window under Emit section below "Emit Dimensions."
- Run this from the console:
```
Entities.addEntity({
	"type": "ParticleEffect",
	"position": MyAvatar.position,
	"isEmitting":true,"lifespan":"1.0","maxParticles":"1000","textures":"https://content.highfidelity.com/DomainContent/production/Particles/wispy-smoke.png","emitRate":"600","emitSpeed":"0","speedSpread":"0","emitDimensions":{"x":"1","y":"1","z":"1"},"emitRadiusStart":"0","emitOrientation":{"x":"-90","y":"0","z":"0"},"emitterShouldTrail":true,"particleRadius":"0.10000000149011612","radiusSpread":"0","radiusStart":"0.10000000149011612","radiusFinish":"0.10000000149011612","color":{"red":"255","blue":"255","green":"255"},"colorSpread":{"red":"0","blue":"0","green":"0"},"colorStart":{"red":"255","blue":"255","green":"255"},"colorFinish":{"red":"255","blue":"255","green":"255"},"emitAcceleration":{"x":"0","y":"0","z":"0"},"accelerationSpread":{"x":"0","y":"0","z":"0"},"alpha":"0.5","alphaSpread":"0","alphaStart":"0.5","alphaFinish":"0.5","particleSpin":0,"spinSpread":0,"spinStart":0,"spinFinish":0,"rotateWithEntity":false,"polarStart":0,"polarFinish":0.7853981633974483,"azimuthStart":-3.1415927410125732,"azimuthFinish":3.1415927410125732
});
```
- You should see something like this:
![image](https://user-images.githubusercontent.com/7650116/47389295-40478680-d6c9-11e8-98c3-3f10971a7e4c.png)
- Edit its "Emit Radius Start".  As it gets closer to 1, it should look more like this:
![image](https://user-images.githubusercontent.com/7650116/47389337-59503780-d6c9-11e8-8a0c-a43317a2f055.png)
